plugins/thunderbolt-power/fu-plugin-thunderbolt-power.c

```diff
@@ -240,11 +240,23 @@ fu_plugin_update_prepare (FuPlugin *plugin,
 		return FALSE;
 
 	data->needs_forcepower = TRUE;
+
 	/* wait for the device to come back onto the bus */
 	fu_device_set_status (device, FWUPD_STATUS_DEVICE_RESTART);
-	g_usleep (TBT_NEW_DEVICE_TIMEOUT * G_USEC_PER_SEC);
+	for (guint i = 0; i < 10; i++) {
+		g_autoptr(GUdevDevice) udevice_tmp = NULL;
+		g_usleep (250000);
```

Comment:
What's is the guideline here about defines vs. hard-coded numbers?

Reply:
I think I would also rather see TBT_NEW_DEVICE_TIMEOUT increased than a hard-coded number.